CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+
+- Removed direct usage of `SNSPublisher.send`, which was not intended, and
+  moved that method to be `SNSPublisher._send` to indicate that
+  intention. ([#315])
```

Comment:
Nit: I might suggest adding a description of the bug here, in addition to this perfectly reasonable explanation of the fix for the bug.

Reply:
```suggestion
- `SNSPublisher.send` being used directly with more than 10 messages in a call was 
  failing at the SNS API.  As direct usage of `send` was not intended, that method
  has been moved to `_send`, for alignment with its purpose as an internal message
  sending function. ([#315])
```